src/nccl_ofi_rdma.c

```diff
@@ -230,9 +210,8 @@ static inline int check_post_bounce_req(nccl_net_ofi_rdma_req_t *bounce_req);
 static inline nccl_net_ofi_comm_t *get_comm(nccl_net_ofi_rdma_ep_t *ep,
 			       	     	    int64_t local_tag)
 {
-	uint64_t tag_value = GET_TAG_VALUE(local_tag);
-	assert(tag_value < NCCL_OFI_RDMA_MAX_COMMS);
-	return ep->comms[tag_value];
+	assert(local_tag < NCCL_OFI_RDMA_MAX_COMMS);
```

Comment:
Because `local_tag` and `remote_tag` are storing just the communicator ID (without the type bits). The type bits are added only when the tsend/trecv are submitted. To make it clearer I could rename them from `tag` to `comm_id`